prime-router/src/main/kotlin/azure/BatchFunction.kt

```diff
@@ -192,7 +194,7 @@ class BatchFunction(
                 actionHistory.trackExistingInputReport(it.task.reportId)
 
                 // download message
-                val bodyBytes = BlobAccess.downloadBlob(it.task.bodyUrl)
+                val bodyBytes = downloadBlobWithMetrics(it)
```

Comment:
This change will measure the time it takes to download a single report, right? What we want to measure is the time it takes to download all reports, correct? For that is what we are aiming to speed up? So, where it says "Batch Download Time: " it should actually say "Blob Download Time" (which we can't speed up and probably shouldn't measure)?

Reply:
Also why are we downloading here again anyways? It seems like validHeaders would already have the bytes in `content` field by now? Populated from `createHeader`?